.nvmrc

```diff
@@ -0,0 +1 @@
+v14.16.0
```

Comment:
Have we seen students using NVM? Because we always want students on the latest stable version of Node, and that changes about every 6 months, I'm hesitant to hardcode that version anywhere. It becomes another thing to remember to update...

Reply:
I don’t think so. It would be purely optional. Maybe we ditch the minor and patch version here and do `v14`?